torchci/rockset/pytorch_dev_infra_kpis/__sql/time_to_signal.sql

```diff
@@ -1,4 +1,27 @@
-with completed_workflows as (
+with failed_workflows as (
```

Comment:
Nit: There is a mixture of lowercase, UPPERCASE, and Camel from the query. I think it would look nicer to stick to one case